docs/concepts/index.md

```diff
@@ -0,0 +1,17 @@
+## Overview
+
+### What is a Generator Tool, anyway?
```

Comment:
```suggestion

```
So I would remove this header because it seems more playful; something I'd see in a blog post for example but not necessarily documentation. 

Also, let's have the first header of `Overview` be the sole header for this introduction bit.